mozilla-release/config/rules.mk

```diff
@@ -1206,14 +1206,18 @@ endif # SDK_BINARY
 # NEW:
 ifdef CQZ_BUILD_ID
 CLIQZ_EXT_URL = "http://repository.cliqz.com/dist/$(MOZ_UPDATE_CHANNEL)/$(CQZ_VERSION)/$(CQZ_BUILD_ID)/cliqz@cliqz.com.xpi"
+HTTPSE_EXT_URL = "http://repository.cliqz.com/dist/$(MOZ_UPDATE_CHANNEL)/$(CQZ_VERSION)/$(CQZ_BUILD_ID)/https-everywhere@cliqz.com.xpi"
 endif  # CQZ_BUILD_ID
 
 # OLD:
 ifndef CQZ_BUILD_ID
 # TODO: Move to external file.
 CLIQZ_EXT_URL = "http://cdn2.cliqz.com/update/browser_beta/latest.xpi"
+HTTPSE_EXT_URL = "https://s3.amazonaws.com/cdncliqz/update/browser_beta/https-everywhere/https-everywhere-5.1.9~5b7adc4-cliqz-signed.xpi"
 ifeq (release, $(CQZ_RELEASE_CHANNEL))
 CLIQZ_EXT_URL = "http://cdn2.cliqz.com/update/browser/Cliqz.1.4.0.xpi"
+HTTPSE_EXT_URL = "https://s3.amazonaws.com/cdncliqz/update/browser_beta/https-everywhere/https-everywhere-5.1.9~5b7adc4-cliqz-signed.xpi"
```

Comment:
it will be difference, once we will create "release" build for https-everywhere. this is a placeholder